config/locales/en.yml

```
@@ -391,6 +391,10 @@ en:
 
   provider:
     passwords:
+      destroy:
+        success: A password reset link has been emailed to you.
```

```suggestion
        success: If this email address is owned by an account, a password reset link has been sent to it.
```

---

This is from sso.redhat.com/auth:
```suggestion
        success: You will receive an email shortly with instructions on how to create a new password. TIP: Check your junk or spam folder if you are unable to find the email.
```

---

```suggestion
        success: We sent an email with password reset instructions to: username@domain.com. If you do not receive it soon, check your spam folder or request a password reset again.
```